wiki_doc

```diff
@@ -0,0 +1,190 @@
+## Valkey Commands
+
+For information on the supported commands and their corresponding parameters, we recommend referring to the documentation in the code. This documentation provides in-depth insights into the usage and options available for each command.
+
+### Command String Arguments
+
+Valkey stores strings as sequences of bytes, so that may include text, serialized objects, or binary arrays. As such, to pass Valkey strings as arguments to commands, or receive Valkey strings in responses, GLIDE offers `GlideString`, which represents two APIs:
```

Comment:
This looks like it's copied from Redis.io https://redis.io/docs/latest/develop/data-types/strings/
We should consider re-writing this first paragraph from the perspective of a Node.js user.  Such as: 
```
Valkey strings are stored in its simplest form: as a sequence of bytes.  GLIDE offers [GlideString](https://github.com/valkey-io/valkey-glide/blob/main/node/src/BaseClient.ts#L278) which maps to   Valkey strings.  Use it to pass Valkey strings as arguments to commands, or receive Valkey strings in response to commands.  GlideString represents either: 
1. `string`: as a UTF-8 encoded string.
2. `Buffer`: as a `byte[]`. 
```